doc/conf.py

```diff
@@ -18,23 +20,23 @@
 
 # -- Project information -----------------------------------------------------
 
-_config = configparser.RawConfigParser()
-_config.read(os.path.join("..", "setup.cfg"))
-_meta = dict(_config.items("metadata"))
+toml_config = toml.load(os.path.join("..", "pyproject.toml"))
+project = toml_config["project"]["name"]
+author = ", ".join([a["name"] for a in toml_config["project"]["authors"]])
```

Comment:
author does not seem to be used

Reply:
It is actually used by sphinx: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-author